mbofui/app/js/controllers.js

```diff
@@ -257,10 +257,8 @@ ocellus.controller('mapController', ['$compile', '$scope', '$rootScope','$filter
     }
     else {
       //populate the bofModalEdit modal and show it
-      thisEvent.startTime = new Date(thisEvent.startTime);
-      thisEvent.endTime = new Date(thisEvent.endTime);
-      $('#bofModalEdit #startTimeEdit').val(moment(thisEvent.startTime).format('YYYY-M-DThh:mm'));
-      $('#bofModalEdit #endTimeEdit').val(moment(thisEvent.endTime).format('YYYY-M-DThh:mm'));
+      $('#bofModalEdit #startTimeEdit').val(moment(thisEvent.startTime).format('YYYY-MM-DDTHH:mm'));
+      $('#bofModalEdit #endTimeEdit').val(moment(thisEvent.endTime).format('YYYY-MM-DDTHH:mm'));
```

Comment:
I see that you are using the string 'YYYY-MM-DDTHH:mm' twice here. Is it possible to make this a variable so that we can simply reuse it in future JIRAs? This way we won't have to worry about getting it exact.